web/models/respondent_stats.ex

```diff
@@ -114,6 +114,6 @@ defmodule Ask.RespondentStats do
 
   def respondents_by_disposition(survey) do
     respondent_count(survey_id: ^survey.id, by: :disposition)
-    |> Enum.into(%{})
+    |> Enum.into(%{}, fn ({k, v}) -> {String.to_existing_atom(k), v} end)
```

Comment:
Maybe we could use:
```
|> Map.new(fn {k, v} -> { String.to_existing_atom(k), v } end)
```
Just to emphasize that we are creating a new `Map` (instead of using `Enum.into(%{}, ...)`

Reply:
@ftarulla I also made `respondents_by_state` above use `Map.new` and `Map.merge` because `Enum.into` is deprecated for non empty enumerables :shrug: 

I also build the default Map from the actual Ecto.Enum. It was missing `:cancelled` for example.